StatusEffect/SexEngineHypnovisor.gd

```diff
@@ -1,80 +1,11 @@
 extends StatusEffectBase
 
-var progress = 0.0
-
 func _init():
-	id = StatusEffect.SexEngineHypnovisor
-	isBattleOnly = false
-	
-	alwaysCheckedForNPCs = true
-	alwaysCheckedForPlayer = true
-	priorityDuringChecking = 48
+	pass
 	
 func shouldApplyTo(_npc):
-	if(GM.main != null && GM.main.supportsSexEngine()):
-		if(_npc.getInventory().hasSlotEquipped(InventorySlot.Eyes) && _npc.getInventory().getEquippedItem(InventorySlot.Eyes).hasTag(ItemTag.Hypnovisor)):
-			return true
+	#disable in favor of Hypnokink's effect
```

Comment:
Might as well remove this file then

Reply:
Had to keep it around for the mod (so It'd override) but could have sworn I deleted it for this branch. The godot editor's been sneakily undeleting files on me for some reason though, so that's probably how it got back in here.

Reply:
Done.